sandy/core/cov.py

```diff
@@ -1962,6 +1958,73 @@ def sparse_tables_inv(a, rows=1000):
     return invert_matrix
 
 
+def segmented_pivot_table(data_stack, index, columns, values, rows=10000000):

```

Comment:
@EnricaBelfiore , do we use this?

Reply:
Yes, it is used in the class method `sandy.CategoryCov.from_stack()`